DonWorry/DonWorry/Helper/ImageManager.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  ImageManager.swift
+//  DonWorry
+//
+//  Created by 임영후 on 2022/06/03.
+//
+
+import Foundation
+import UIKit
+
+class ImageManager {
+    
+    static let instance = ImageManager()
+    
+    private init() { }
+    
+    func saveImage(uiImage: UIImage) -> String {
+        let imageData = uiImage.jpegData(compressionQuality: 0.5)
+        guard let imageData = imageData else { return "" }
+        
+        let url = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask)[0]
+        let filName = "\(UUID().uuidString).jpeg"
+        let filePath = url.appendingPathComponent(filName)
+        
+        do {
+            try imageData.write(to: filePath)
+        } catch {
+            print(error.localizedDescription)
+        }
+        
+        return filName
```

Comment:
[0]으로 접근하게 되면 배열에 아무것도 없을 때 예상치 못한 오류가 나요. 그래서 옵셔널이라로 접근하는 게 좋은 방식 같습니다!

```suggestion
      if let url = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first {
            let filName = "\(UUID().uuidString).jpeg"
            let filePath = url.appendingPathComponent(filName)
            
            do {
                try imageData.write(to: filePath)
            } catch {
                print(error.localizedDescription)
            }
            return filName
        }
        return ""
```

Reply:
ㄱㅅ